pages/search/advanced.tsx

```diff
@@ -168,63 +199,59 @@ export default function AdvancedSearch({
           onSubmit={handleSubmit}
         >
           <Flex flexDirection={{ base: "column", md: "row" }}>
-            <Flex id="advancedSearchLeft" gap="s" direction="column" grow="1">
-              {textInputFields.map(({ name, label }) => {
-                return (
-                  <FormField key={name}>
-                    <TextInput
-                      id={name}
-                      labelText={label}
-                      name={name}
-                      value={searchFormState[name]}
-                      onChange={debounce(
-                        (e) => handleInputChange(e, "input_change"),
-                        DEBOUNCE_INTERVAL
-                      )}
-                      ref={inputRef}
-                    />
-                  </FormField>
-                )
-              })}
+            <Flex
+              id="advancedSearchLeft"
+              gap="s"
+              direction="column"
+              grow="1"
+              width={{ base: "100%", md: "50%" }}
+            >
+              {textInputFields.map(({ name, label }) => (
+                <FormField key={name}>
+                  <TextInput
+                    id={name}
+                    labelText={label}
+                    name={name}
+                    value={searchFormState[name]}
+                    onChange={debounce(
+                      (e) => handleInputChange(e, "input_change"),
+                      DEBOUNCE_INTERVAL
+                    )}
+                  />
+                </FormField>
+              ))}
+
               <FormField>
-                <Select
-                  id="languageSelect"
-                  name="language"
-                  labelText="Language"
-                  value={searchFormState["filters"].language}
-                  onChange={(e) => handleInputChange(e, "filter_change")}
-                >
-                  {languageOptions.map((language) => {
-                    return (
-                      <option value={language.value} key={language.value}>
-                        {language.label}
-                      </option>
-                    )
-                  })}
-                </Select>
+                <DateFilter {...dateFilterProps} />
               </FormField>
-              <FormField>{<DateFilter {...dateFilterProps} />}</FormField>
             </Flex>
-            <Flex direction="column" gap="l" grow="1">
-              <SearchFilterCheckboxField
-                options={searchVocabularies.buildingLocations}
-                name="location"
-                label="Item location"
-                handleCheckboxChange={(e) =>
-                  handleCheckboxChange("buildingLocation", e)
+            <Flex
+              direction="column"
+              gap={{ base: "m", md: "43px" }}
+              grow="1"
+              mt="m"
+              width={{ base: "100%", md: "50%" }}
+            >
+              {multiselects}
+              <GroupedMultiSelect
```

Comment:
This is a nit, but i think it might make sense to include the `groupedMultiSelect` in the multiselects array.